src/NServiceBus.AzureFunctions.Worker.SourceGenerator/TriggerFunctionGenerator.cs

```diff
@@ -60,6 +75,7 @@ public void OnVisitSyntaxNode(GeneratorSyntaxContext context)
                 bool IsNServiceBusEndpointNameAttribute(string value) => value?.Equals("NServiceBus.NServiceBusTriggerFunctionAttribute") ?? false;
                 string AttributeParameterAtPosition(int position) => context.SemanticModel.GetConstantValue(attributeSyntax.ArgumentList.Arguments[position].Expression).ToString();
                 int AttributeParametersCount() => attributeSyntax.ArgumentList.Arguments.Count;
+                bool IsBindingExpression(string endpointName) => endpointName[0] == endpointName[endpointName.Length - 1] && endpointName[0] == '%';
```

Comment:
wouldn't this still require some null check? e.g.

```suggestion
                bool IsBindingExpression(string endpointName) => endpointName == null ? fasle : (endpointName[0] == endpointName[endpointName.Length - 1] && endpointName[0] == '%');
```